js/internal-template-events.js

```diff
@@ -1,15 +1,59 @@
-(function ($) {
-  function ratePage() {
-    var urlElm = document.getElementById('url');
-    
-    if (urlElm) {
-        urlElm.value = window.location.href;
-    }
-
-    if ($('input#website').val().length) {
-      return false;
-    }
-  }
-  /*Submit url to rate form*/
-  $(document).on('submit', '#RateThisPageForm', ratePage);
-})(jQuery);
\ No newline at end of file
+(function onTemplateEventsInit($) {
+	var resizeTimer;
+	var windowWidth;
+
+	function $getSearchContainer(width) {
+		return isMobile(width)
+			? $('#search-container')
+			: $('#internal-search-container');
+	}
+
+	function isMobile(width) {
+		var mediaWidth = 990;
```

Comment:
This introduces a logical dependency on a specific breakpoint. Granted, those don't change often.

Reply:
@mxsnyder this part of the logic was unchanged form the original code